### ISSUE TITLE
sass line comment not support

### ISSUE THREAD

**Issue by zhylmzr** [farm-fe/farm#909]
### Steps to reproduce

Create a vue project, and in App.vue style:

```vue
<style lang="scss" scoped>
body {
    // background: red;
}
</style>
```

### What is expected?

work fine

### What is actually happening?

```
[ Farm ] Error: {"name":"CssSyntaxError","reason":"Unknown word","file":".../src/App.vue","source":"\nbody {\n    // background: red;\n}\n","line":3,"column":8,"endLine":3,"endColumn":18,"input":{"column":8,"endColumn":18,"endLine":3,"line":3,"source":"\nbody {\n    // background: red;\n}\n","url":"file://.../src/App.vue","file":".../src/App.vue"},"loc":{"file":".../src/App.vue","line":14,"column":8},"code":"PLUGIN_ERROR","plugin":"vite:vue","id":"src/App.vue?vue&type=style&index=0&scoped=7a7a37b1&lang.scss","hook":"transform"}
```

### System Info

```Shell
not important
```

**Comment by wre232114** [comment]
Do you install plugin `@farmfe/plugin-sass`?

**Comment by zhylmzr** [comment]
I've added the reproducible [repository](https://github.com/zhylmzr/farm-sass-bug)

**Comment by wre232114** [comment]
fixed in `@farmfe/core@0.16.0`

**Comment by zhylmzr** [comment]
thanks
